nixos/modules/services/hardware/nvidia-container-toolkit/cdi-generate.nix

```diff
@@ -32,6 +35,18 @@ function cdiGenerate {
     --nvidia-ctk-path ${lib.getExe' nvidia-container-toolkit "nvidia-ctk"}
 }
 
-cdiGenerate | \
-  ${lib.concatStringsSep " | " allJqMounts} > $RUNTIME_DIRECTORY/nvidia-container-toolkit.json
+function additionalMount {
```

Comment:
Well one comment is that this could be rewritten with `pythonMinimal` or a similar language that doesn't try so much to make this painful

Reply:
But as commented before, if you say this is the current iteration I'll go with it

Reply:
Yes, I'll work on a reimplementation of this logic with Python in future PR's, if that's fine with you.